server/task_executor_database_create.go

```diff
@@ -120,6 +120,57 @@ func (exec *DatabaseCreateTaskExecutor) RunOnce(ctx context.Context, server *Ser
 		return true, nil, err
 	}
 
+	// After the database creation statement executed rightly,
```

Comment:
```suggestion
	// After the database creation statement executed successfully,
```

Reply:
It's still worthwhile to move some comments here and may add additional comments 

"	// If create database task completes, then we will create a database entry immediately
	// instead of waiting for the next schema sync cycle to sync over this newly created database.
	// This is for 2 reasons:
	// 1. Assign the proper project to the newly created database. Otherwise, the periodic schema
	//    sync will place the synced db into the default project.
	// 2. Allow user to see the created database right away."